src/main/java/seedu/address/logic/commands/AddSellerCommand.java

```diff
@@ -26,31 +30,39 @@ public class AddSellerCommand extends Command {
             + PREFIX_NAME + "NAME "
             + PREFIX_PHONE + "PHONE "
             + PREFIX_EMAIL + "EMAIL "
-            + PREFIX_ADDRESS + "ADDRESS "
+            + PREFIX_HOUSINGTYPE + "HOUSINGTYPE "
+            + PREFIX_STREET + "STREET "
+            + PREFIX_BLOCK + "BLOCK "
+            + PREFIX_LEVEL + "LEVEL "
+            + PREFIX_UNITNUMBER + "UNIT NUMBER "
             + PREFIX_POSTALCODE + "POSTAL CODE "
             + "[" + PREFIX_TAG + "TAG]...\n"
             + "Example: " + COMMAND_WORD + " "
             + PREFIX_NAME + "John Doe "
             + PREFIX_PHONE + "98765432 "
             + PREFIX_EMAIL + "johnd@example.com "
-            + PREFIX_ADDRESS + "311, Clementi Ave 2, #02-25 "
+            + PREFIX_HOUSINGTYPE + "HDB "
+            + PREFIX_STREET + "Clementi Ave 2 "
+            + PREFIX_BLOCK + "311 "
+            + PREFIX_LEVEL + "02 "
+            + PREFIX_UNITNUMBER + "25 "
             + PREFIX_POSTALCODE + "578578 "
             + PREFIX_TAG + "friends "
-            + PREFIX_TAG + "owesMoney";
+            + PREFIX_TAG + "owesMoney ";
 
-    public static final String MESSAGE_SUCCESS = "New seller added: %1$s";
-    public static final String MESSAGE_DUPLICATE_PERSON = "This seller already exists in the address book";
+    public static final String MESSAGE_SUCCESS = "New seller added= %1$s";
+    public static final String MESSAGE_DUPLICATE_SELLER = "This seller already exists in the address book";
```

Comment:
Need to change the "in the address book" to "in EstateEase"

Reply:
Okay, will do so :)